src/test/java/hudson/plugins/im/bot/HealthCommandTest.java

```diff
@@ -42,6 +43,7 @@ public void testNoJobFound() {
 
     @SuppressWarnings("rawtypes")
     @Test
+    @Ignore("Needs rewriting to not use mockito")
```

Comment:
these sort of tests don't work on modern jenkins because of some bytecoding patching that's done for compatibly mockito can't understand it and it means if you have a test that touches this are it's impossible to write with mockito.

Reply:
Just to understand clearer through (I think) typos:

> these sort of tests don't work on modern jenkins because of some bytecoding patching that's done for compatibility. mockito can't understand it, and it means if you have a test that touches this area it's impossible to write with mockito.

Reply:
yes